python/src/spark_rapids_ml/umap.py

```diff
@@ -139,6 +139,21 @@ def setLabelCol(self: P, value: str) -> P:
         Sets the value of :py:attr:`labelCol`.
         """
         return self.set_params(labelCol=value)
+    
+    def getOutputCol(self) -> str:
+        """
+        Gets the value of :py:attr:`outputCol`. Contains the embeddings of the input data.
+        """
+        if self.isDefined(self.outputCol):
+            return self.getOrDefault("outputCol")
```

Comment:
Seems no need to check whether the outputCol has been configured. It's ok to just return self.getOrDefault("outputCol")